src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -387,8 +387,18 @@ def _handleNoStatusUpdate(self, workload, request_args):
 
     def _handleAssignmentStateTransition(self, workload, request_args, dn):
         
+        req_status = request_args["RequestStatus"]
+        
+        if not request_args.get('Team', '').strip() and \
+           not request_args.get('Teamd', []) and req_status == "assigned":
```

Comment:
Thanks I will fix that. Yes args get from web interface and I set only for Team not Teams so we should be fine for now.